client/quantum_serverless/core/provider.py

```diff
@@ -584,5 +584,5 @@ def _fetch_token(self, username: str, password: str):
         if not gateway_response.ok:
             raise QuantumServerlessException(gateway_response.text)
 
-        gateway_token = json.loads(gateway_response.text).get("access_token")
+        gateway_token = json.loads(gateway_response.text).get("access")
```

Comment:
Oh, when did the `access_token` keyword change to `access`?

Reply:
One of the dependencies released a new major version which got pulled in for the latest builds. @IceKhan13 suggested we may want to pin the deps in #509 , +1 for me.

Reply:
Basically this comment https://github.com/Qiskit-Extensions/quantum-serverless/pull/509#issuecomment-1538686258

Reply:
Yes! I couldn't answer before due to the problems in GitHub haha. I skipped that comment, thank you team 😄